src/main/java/org/interledger/InterledgerAddress.java

```diff
@@ -33,6 +33,7 @@
    * Constructor to allow quick create of String.
```

Comment:
Hey @controlsurface Thanks for submitting this PR! Just some minor nits, but overall looks great!

Reply:
@sappenin Thank you for the comments :-) I have corrected the issues.